src/components/CountryCard.module.scss

```diff
@@ -46,8 +46,8 @@
 
   .card {
     min-width: 180px;
-    width: 180px;
-    height: 220px;
+    width: 220px;
+    height: 260px;
```

Comment:
Try to use css grid to solve card list layout.

Reply:
a good idea